apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -93,80 +82,47 @@ public async Task BadDbName()
             string connectionString = new SpannerConnectionStringBuilder(_fixture.ConnectionString)
                 .WithDatabase("badjuju")
                 .ConnectionString;
-            int rowsRead = -1;
 
-            var e = await Assert.ThrowsAsync<SpannerException>(
-                async () =>
-                {
                     using (var connection = new SpannerConnection(connectionString))
                     {
-                        var cmd = connection.CreateSelectCommand(
-                            $"SELECT * FROM {_fixture.TableName} WHERE Key = 'k1'");
-                        using (var reader = await cmd.ExecuteReaderAsync())
-                        {
-                            rowsRead = 0;
-                            while (await reader.ReadAsync())
-                            {
-                                rowsRead++;
-                            }
-                        }
-                    }
-                }).ConfigureAwait(false);
-
+                var cmd = connection.CreateSelectCommand($"SELECT * FROM {_fixture.TableName} WHERE Key = 'k1'");
+                var e = await Assert.ThrowsAsync<SpannerException>(() => cmd.ExecuteReaderAsync());
             Assert.Equal(ErrorCode.NotFound, e.ErrorCode);
             Assert.False(e.IsTransientSpannerFault());
-            Assert.Equal(-1, rowsRead);
+            }
         }
 
         [Fact]
         public async Task BadTableName()
-        {
-            int rowsRead = -1;
-
-            var e = await Assert.ThrowsAsync<SpannerException>(
-                async () =>
                 {
                     using (var connection = _fixture.GetConnection())
                     {
                         var cmd = connection.CreateSelectCommand("SELECT * FROM badjuju WHERE Key = 'k99'");
                         using (var reader = await cmd.ExecuteReaderAsync())
                         {
-                            rowsRead = 0;
-                            while (await reader.ReadAsync())
-                            {
-                                rowsRead++;
-                            }
-                        }
-                    }
-                }).ConfigureAwait(false);
-
+                    var e = await Assert.ThrowsAsync<SpannerException>(() => reader.ReadAsync());
```

Comment:
nit: In general indentation is off on this whole file.

On the other side, +100 on rewriting the tests this way.